### DISCUSSION TITLE
Why can I not create a new private repo in the organization with the Free plan?

### DISCUSSION THREAD

**Discussion by palbers97** [community/community#48801] [Repositories] [answered]
### Select Topic Area

Question

### Body

When I want to create a new repository I cannot select _Private_. It says:

> Your current plan does not support private repositories. Your organization's owners will need to upgrade to Team.

How does this fit together with the statement on the _Pricing_ page that orgas in the free plan can create unlimited private repos?

Thanks in advance!

**Comment by Anmol-Baranwal** [comment]
There may be a error on their side. I would urge you to contact [support](https://support.github.com/request)

> Pin me if you have any further doubts.

**Comment by Anmol-Baranwal** [comment]
@palbers97 
The statement on the Pricing page that organizations in the free plan can create unlimited private repositories is correct. However, there is a limit on the number of users who can access private repositories on the free plan.

If your organization has more than three users, then you will need to upgrade to the Team plan to create private repositories that are accessible to all members of your organization. The free plan only allows for three users to access private repositories, while the Team plan allows for unlimited users.

If you only need to create private repositories for yourself or for a small team of up to three people, then you can continue to use the free plan and create private repositories that are only accessible to those specific users.

**Reply by Anmol-Baranwal** [reply]
@palbers97 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.

**Reply by palbers97** [reply]
Thanks for your help!

**Comment by palbers97** [comment, marked as answer]
In the end, the support answered. The reason was that our organization has an old legacy Gold plan with unlimited users but private repositories limited to 50.

**Reply by Anmol-Baranwal** [reply]
Thanks for letting us know.
